client/app/account/components/AccountInfo.tsx

```diff
@@ -2,15 +2,19 @@ import React from 'react'
 import { MdAccountCircle } from 'react-icons/md'
 import { Loading } from '@/_components/layout'
 // import { RouterLink } from '@/_components/ui'
+import { RouterLink } from '@/_components/ui'
```

Comment:
Could you delete the 4th line? Is it duplicated?

Reply:
Deleted it. 
e91b6ea84997bbfee8778a340fd7258b1696eee1